gecoscc-installer.sh

```diff
@@ -177,38 +178,44 @@ NO)
   esac
 fi
 
-
-echo "Adding EPEL repository"
+#echo "Adding EPEL repository"
 if ! rpm -q epel-release-6-8.noarch; then
-    rpm -ivh http://dl.fedoraproject.org/pub/epel/6/x86_64/epel-release-6-8.noarch.rpm
+#if ! rpm -q epel-release-7-9.noarch; then
+    rpm -ivh --nosignature http://dl.fedoraproject.org/pub/epel/6/x86_64/epel-release-6-8.noarch.rpm
+    #rpm -ivh --nosignature https://dl.fedoraproject.org/pub/epel/7/x86_64/e/epel-release-7-9.noarch.rpm
 fi
-echo "Installing python-devel and pip"
-install_package python-devel 
-install_package python-pip
+echo "Installing python2.7"
+yum install -y centos-release-SCL
```

Comment:
Changed.

It appeared this way in the installation procedure and didn't want to change it, although you are right, `install_package` funtion must be used in all the cases.